parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -96,11 +96,13 @@ write_files:
     ExecStartPre=/usr/bin/docker pull {{WrapAsVariable "kubernetesHyperkubeSpec"}}
     ExecStartPre=/usr/bin/docker run --rm -v /tmp/kubectldir:/opt/kubectldir {{WrapAsVariable "kubernetesHyperkubeSpec"}} /bin/bash -c "cp /hyperkube /opt/kubectldir/"
 {{if .IsCoreOS}}
+    ExecStartPre=/bin/cp /tmp/kubectldir/hyperkube /opt/kubelet
     ExecStartPre=/bin/mv /tmp/kubectldir/hyperkube /opt/kubectl
-    ExecStart=/bin/chmod a+x /opt/kubectl
+    ExecStart=/bin/chmod a+x /opt/kubelet && /bin/chmod a+x /opt/kubectl
```

Comment:
See below. Didn't verify on a coreos host, but I'm confident this is a standard pattern.